cfme/fixtures/cli.py

```diff
@@ -74,47 +83,414 @@ def dedicated_db_appliance(app_creds, unconfigured_appliance):
+        ).success
+
+    wait_for(
+        is_ha_monitor_started, func_args=[apps2], timeout=300, handle_exception=True
+    )
+    # Add infra/cloud providers and create db backup
+    provider_app_crud(VMwareProvider, apps2).setup()
+    provider_app_crud(EC2Provider, apps2).setup()
+    return ha_appliances_with_providers
+
+
+@pytest.fixture
+def replicated_appliances_with_providers(multiple_preupdate_appliances):
+    """Returns two database-owning appliances, configures with providers."""
+    appl1, appl2 = multiple_preupdate_appliances
+    # configure appliances
+    appl1.configure(region=0)
+    appl1.wait_for_web_ui()
+    appl2.configure(region=99)
+    appl2.wait_for_web_ui()
+    # configure replication between appliances
+    appl1.set_pglogical_replication(replication_type=":remote")
+    appl2.set_pglogical_replication(replication_type=":global")
+    appl2.add_pglogical_replication_subscription(appl1.hostname)
+    # Add infra/cloud providers
+    provider_app_crud(VMwareProvider, appl1).setup()
+    provider_app_crud(EC2Provider, appl1).setup()
+    return multiple_preupdate_appliances
+
+
+@pytest.fixture
+def ext_appliances_with_providers(multiple_preupdate_appliances, app_creds_modscope):
+    """Returns two database-owning appliances, configures first appliance with providers."""
+    appl1, appl2 = multiple_preupdate_appliances
+    app_ip = appl1.hostname
+    # configure appliances
+    appl1.configure(region=0)
+    appl1.wait_for_web_ui()
+    appl2.appliance_console_cli.configure_appliance_external_join(
+        app_ip,
+        app_creds_modscope["username"],
+        app_creds_modscope["password"],
+        "vmdb_production",
+        app_ip,
+        app_creds_modscope["sshlogin"],
+        app_creds_modscope["sshpass"],
+    )
+    appl2.wait_for_web_ui()
+    # Add infra/cloud providers and create db backup
+    provider_app_crud(VMwareProvider, appl1).setup()
+    provider_app_crud(EC2Provider, appl1).setup()
+    return multiple_preupdate_appliances
+
+
+@pytest.fixture
+def enabled_embedded_appliance(appliance_preupdate):
+    """Takes a preconfigured appliance and enables the embedded ansible role"""
+    appliance_preupdate.enable_embedded_ansible_role()
+    assert appliance_preupdate.is_embedded_ansible_running
+    return appliance_preupdate
+
+
+@pytest.fixture
+def appliance_with_providers(appliance_preupdate):
+    """Adds providers to appliance"""
+    appl1 = appliance_preupdate
+    # Add infra/cloud providers
+    provider_app_crud(VMwareProvider, appl1).setup()
+    provider_app_crud(EC2Provider, appl1).setup()
+    return appliance_preupdate
+
+
+@pytest.fixture(scope="module")
+def ansible_repository(appliance):
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        fauxfactory.gen_alpha(), REPOSITORIES[0], description=fauxfactory.gen_alpha()
+    )
+    view = navigate_to(repository, "Details")
+    if appliance.version < "5.9":
```

Comment:
No need to check for version less than 5.9 .
We are only doing 5.9 and 5.10 now